tests/test_dynamo/test_subscriptions.py

```diff
@@ -214,3 +214,38 @@ def test_put_subscription_update(tables):
 
     response = tables.subscriptions_table.scan()
     assert response['Items'] == [updated_subscription]
+
+
+def test_put_subscription_validate_only(tables):
+    subscription = {
+        'job_definition': {
+            'job_type': 'RTC_GAMMA',
+            'name': 'sub1',
+        },
+        'search_parameters': {
+            'start': '2020-01-01T00:00:00+00:00',
+            'end': '2020-01-01T00:00:00+00:00',
+        }
+    }
+    with pytest.raises(ValueError):
+        dynamo.subscriptions.put_subscription('user1', subscription, validate_only=True)
```

Comment:
Checking for ValueError when validate_only=False is covered elsewhere, yes?

Reply:
looks like ValueError is only checked for in `test_validate_subscription()`, and `validate_subscription()` doesn't know about validate_only, so I'd err on the side of adding the two exra lines to do the check here

Reply:
so nothing actionable?

Reply:
I'd add
```
    with pytest.raises(ValueError):
        dynamo.subscriptions.put_subscription('user1', subscription, validate_only=False)
```